lib/db/StudyRequestDAO.js

```diff
@@ -99,6 +99,24 @@ ORDER BY "id" ASC`;
     return studyRequestsSchema.validateAsync(studyRequests);
   }
 
+  /**
+   * Fetch study requests that are part of one of the given bulk study requests.
+   *
+   * @param {Array} studyRequestsBulk - bulk study request objects
+   * @returns {Array<Object>} array of study request objects that are part of the given
+   * bulk study request
+   */
+  static async byStudyRequestsBulk(studyRequestsBulk) {
```

Comment:
maybe make this only fetch "pending" (i.e. non-closed) requests as well?  that way we can use results from this to determine which bulk requests are pending...